inc/html.class.php

```diff
@@ -5666,7 +5666,9 @@ static function file($options = []) {
          $display .= "<input id='fileupload{$p['rand']}' type='file' name='".$p['name']."[]'
                          data-url='".$CFG_GLPI["root_doc"]."/ajax/fileupload.php'
                          data-form-data='{\"name\": \"".$p['name']."\",
-                                          \"showfilesize\": \"".$p['showfilesize']."\"}'".($p['multiple']?" multiple='multiple'":"").">";
+                                          \"showfilesize\": \"".$p['showfilesize']."\"}'"
+                         .($p['multiple']?" multiple='multiple'":"")
+                         .($p['onlyimages']?" accept='.gif,.png,.jpg,.jpeg'":"").">";
```

Comment:
I propose use this.
```suggestion
                         .($p['onlyimages']?" accept='image/*'":"").">";
```

![image](https://user-images.githubusercontent.com/33253653/104428922-53ceb880-5585-11eb-986e-acb5821f8393.png)

Reply:
I now about this way, but the problem this way accept svg images, and them are not allowed to upload